src/app/layout/Page/index.tsx

```diff
@@ -47,43 +50,67 @@ const PageContainer = ({ children, ...rest }) => {
 type PageTopBarProps = FlexProps & {
   onBack?(): void;
   showBack?: boolean;
+  isFixed?: boolean;
 };
 
 export const PageTopBar = ({
   children,
   onBack = () => undefined,
   showBack = false,
+  isFixed = false,
   ...rest
 }: PageTopBarProps) => {
+  const topBarRef = useRef<HTMLDivElement>(null);
+  const [height, setHeight] = useState(0);
+  const { isFocusMode } = useLayoutContext();
+  const theme = useTheme();
+
+  useLayoutEffect(() => {
+    const onResize = () => {
```

Comment:
You should not register resize event if the PageTopBar is not fixed 😉

Reply:
fixed